src/components/TodaySection.astro

```diff
@@ -92,6 +92,7 @@ const subtitles = [
 	let showTodatSectionTriggerScroll = 0
 	const bleachSpeedMS = 50
 	const bleachStep = 0.05
+	let flashInProgress = false
```

Comment:
Please check this case:
scroll to the beginning of the "flash" animation, and right after it's started, start scrolling back until flash animation ends. And then scroll forward.

Reply:
Fixed